Source/WebKit/GPUProcess/GPUConnectionToWebProcess.h

```diff
@@ -336,6 +336,7 @@ class GPUConnectionToWebProcess
     void didReceiveInvalidMessage(IPC::Connection&, IPC::MessageName) final;
     void didReceiveMessage(IPC::Connection&, IPC::Decoder&) final;
     bool didReceiveSyncMessage(IPC::Connection&, IPC::Decoder&, UniqueRef<IPC::Encoder>&) final;
+    bool requestRemoteProcessTermination() final { return false; }
```

Comment:
I thought you told me earlier that there was a single subclass? Now it looks a lot worse since there are many subclasses which basically provide a dummy implementation, which could be defined on the base class instead. This was a step in the wrong direction.

Reply:
Ah, that is a good point, Chris! I am going back to having the dummy implementation in the base class.

Thanks for reviewing!